src/utilities/spacing/spacing.scss

```diff
@@ -41,11 +48,18 @@
     }
   }
 
+  /**
+    * 1. @TODO - Re-evaluate shorthand properties if they become fully supported
```

Comment:
Same feedback as the other comment:

- [ ] Remove `@TODO`: There's nothing actionable here until something changes in browsers, so this is just extra noise.
- [ ] Remove the commented out shorthand values, they are also just noise.
- [ ] Revise the comment to be about _why_ we're using longhand values and not about _what_ we should do later. Example: "Ideally we could use the `margin-inline` shorthand, but it is not as well supported in browsers."